src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -84,16 +86,28 @@
     private static final String TAG_LUMBERJACK_STATISTICS = "lumberjackStatistics";
     private static final String TAG_LUMBERJACK_TREES      = "trees";
     private static final String TAG_LUMBERJACK_SAPLINGS   = "saplings";
-    private static final int    NUM_ACHIEVEMENT_FIRST    = 1;
+    private static final int    NUM_ACHIEVEMENT_FIRST     = 1;
     private static final int    NUM_ACHIEVEMENT_SECOND    = 25;
     private static final int    NUM_ACHIEVEMENT_THIRD     = 100;
     private static final int    NUM_ACHIEVEMENT_FOURTH    = 500;
     private static final int    NUM_ACHIEVEMENT_FIFTH     = 1000;
+    private static final int    CITIZEN_MINIMUM_FOR_RAID  = 5;
+    private static final int    NUM_ONE                   = 1;
+
+    /**
+     * Boolean for whether a raid event has executed this night
+     */
+    private boolean nightHasHappened = true;
+
+    /**
+     * Boolean value for whether a raid will execute the next night
+     */
+    public boolean raidWillHappen = true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make raidWillHappen a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)